packages/trader/src/Modules/Trading/Components/Form/TradeParams/Multiplier/cancel-deal.tsx

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
 
 import { Checkbox, Dropdown, Popover, PopoverMessageCheckbox } from '@deriv/components';
-import { observer, useStore } from '@deriv/stores';
-import { localize } from '@deriv/translations';
-
+import { Localize, localize } from '@deriv/translations';
 import Fieldset from 'App/Components/Form/fieldset';
-import { onChangeCancellationDuration, onToggleCancellation } from 'Stores/Modules/Trading/Helpers/multiplier';
+import { onToggleCancellation, onChangeCancellationDuration } from 'Stores/Modules/Trading/Helpers/multiplier';
```

Comment:
let's not change this line (keep it as it is on master) alphabetically🙏🏻

Reply:
Done, thank you